flow360/component/flow360_params/flow360_params.py

```diff
@@ -706,9 +706,9 @@ class _FluidProperties(Flow360BaseModel):
     """
 
     temperature: TemperatureType = pd.Field()
-    pressure: PressureType = pd.Field()
-    density: DensityType = pd.Field()
-    viscosity: ViscosityType = pd.Field()
+    pressure: PressureType.NonNegative = pd.Field()
+    density: DensityType.NonNegative = pd.Field()
+    viscosity: ViscosityType.NonNegative = pd.Field()
```

Comment:
I guess 0 would not work. All should be positive

Reply:
Correct, fixed to support conversion